management/hook_secrets.go

```diff
@@ -0,0 +1,40 @@
+package management
+
+type HookSecrets = map[string]string
+
+type HookSecretsManager struct {
+	*Management
+}
+
+func newHookSecretsManager(m *Management) *HookSecretsManager {
+	return &HookSecretsManager{m}
+}
+
+// Upserts hook secrets
+//
+// See: https://auth0.com/docs/api/management/v2#!/Hooks/post_secrets
+func (m *HookSecretsManager) Upsert(hookId string, r *HookSecrets) (err error) {
+	return m.post(m.hookPath(hookId), r)
+}
+
+// Reads hook secrets
+//
+// Note: For security, hook secret values cannot be retrieved outside rule
+// execution (they all appear as "_VALUE_NOT_SHOWN_")
+//
+// See: https://auth0.com/docs/api/management/v2#!/Rules_Configs/get_rules_configs
+func (m *HookSecretsManager) Read(hookId string) (r *HookSecrets, err error) {
```

Comment:
When moved to the `HookManager` this could be called `Secrets`.

Reply:
Done